doc/example-nginx-bisontrails.conf

```diff
@@ -0,0 +1,43 @@
+# generate YOUR_BASE64_USERPASS with `echo -n user:password | base64`
```

Comment:
```suggestion
# generate YOUR_BASE64_USERPASS with `echo -n user:password | base64 -w0`
```

Probably want to do this just in case. It will wrap on occasion and I already asked BT to update their docs on this when my first one wrapped.